internal/handlers/scan_sbom_test.go

```diff
@@ -22,54 +26,100 @@ import (
+
 	for _, test := range []struct {
 		platform           string
+		vexHubList         []v1alpha1.VEXHub
 		sourceSBOMJSON     string
 		expectedReportJSON string
 	}{
 		{
 			platform:           "linux/amd64",
+			vexHubList:         []v1alpha1.VEXHub{},
 			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-amd64.spdx.json"),
 			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-amd64.sarif.json"),
 		},
 		{
 			platform:           "linux/arm/v6",
+			vexHubList:         []v1alpha1.VEXHub{},
 			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-arm-v6.spdx.json"),
 			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-arm-v6.sarif.json"),
 		},
 		{
 			platform:           "linux/arm/v7",
+			vexHubList:         []v1alpha1.VEXHub{},
 			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-arm-v7.spdx.json"),
 			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-arm-v7.sarif.json"),
 		},
 		{
 			platform:           "linux/arm64/v8",
+			vexHubList:         []v1alpha1.VEXHub{},
 			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-arm64-v8.spdx.json"),
 			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-arm64-v8.sarif.json"),
 		},
 		{
 			platform:           "linux/386",
+			vexHubList:         []v1alpha1.VEXHub{},
 			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-386.spdx.json"),
 			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-386.sarif.json"),
 		},
 		{
 			platform:           "linux/ppc64le",
+			vexHubList:         []v1alpha1.VEXHub{},
 			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-ppc64le.spdx.json"),
 			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-ppc64le.sarif.json"),
 		},
 		{
 			platform:           "linux/s390x",
+			vexHubList:         []v1alpha1.VEXHub{},
+			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-s390x.spdx.json"),
+			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-s390x.sarif.json"),
+		},
+		{
+			platform: "linux/s390x with VEX repo enabled",
+			vexHubList: []v1alpha1.VEXHub{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "test",
+					},
+					Spec: v1alpha1.VEXHubSpec{
+						URL:     vexHubServer.URL,
+						Enabled: true,
+					},
+				},
+			},
+			sourceSBOMJSON:     filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-s390x.spdx.json"),
+			expectedReportJSON: filepath.Join("..", "..", "test", "fixtures", "golang-1.12-alpine-s390x.sarif.json"),
+		},
+		{
+			platform: "linux/s390x with VEX repo not enabled",
+			vexHubList: []v1alpha1.VEXHub{
```

Comment:
Could we simplify this by adding a vexHub variable directly to the test case, set to nil when there's no VEXHub, and assigned a value when we want to test with on, instead of using a full list?

Reply:
never mind, we agreed to leave it like it is for simplicity